github/commands/pull_request.py

```diff
@@ -121,12 +124,48 @@ def create_branch_for_pr(self, branch_name):
             self.pr["head"]["ref"]
         )
 
+        if not branch_name:
+            branch_name = self.pr["head"]["sha"]
+
```

Comment:
We did not catch `not branch_name` before but here in that case the user did not enter a branch name, and we should probably return/abort the operation.